timemachine/fe/mle.py

```diff
@@ -104,3 +104,96 @@ def loss(x):
 
     # arbitrary additive constant
     return result - result[0]
+
+
+def _bootstrap_node_vals(
+    edge_idxs,
+    edge_diffs,
+    edge_stddevs,
+    ref_node_idxs,
+    ref_node_vals,
+    ref_node_stddevs,
```

Comment:
note: in contrast to https://github.com/proteneer/timemachine/pull/399 , this function does not support a mix of relative and absolute predictions as input.

(absolute references are assumed to be from experiment, and only influence the result via an additive offset regardless the number of references)